Android/app/src/main/java/io/github/project_travel_mate/login/LoginView.java

```diff
@@ -24,7 +24,7 @@
 
     void openSignUp();
 
-    void openLogin();
+    void openLogin(boolean isPassReseted);
```

Comment:
change this variable name to "showToastMessage"

Reply:
done, please re check it